src/moa/README.md

```diff
@@ -0,0 +1,50 @@
+# MOAlmonac
+
+Load variant assestions from the [Molecular Oncology Almonac](https://moalmanac.org)[^1].
```

Comment:
sp - Almonac -> Almanac

Reply:
wow, i did that in a lot of places and didn't even notice :O, will fix